### PR TITLE
Print what was attempted to be exec'd on exec failure

### PR DESCRIPTION
**Pull request by ericcurtin**
Otherwize one must read the code to figure out exactly what exec failed

### CODE REVIEW

**Comment by ericcurtin** [comment]
Now on failure we get something like this.

```
$ ramalama --nocontainer serve granite-code:3b
os.execvp(llama-server, ['llama-server', '--port', '8080', '-m', '/home/curtine/.local/share/ramalama/models/ollama/granite-code:3b', '-c', '2048']) failed
Traceback (most recent call last):
  File "/home/curtine/git/ramalama/./ramalama.py", line 35, in <module>
    main(sys.argv[1:])
  File "/home/curtine/git/ramalama/./ramalama.py", line 20, in main
    ramalama.init_cli()
  File "/home/curtine/git/ramalama/ramalama/cli.py", line 82, in init_cli
    args.func(args)
  File "/home/curtine/git/ramalama/ramalama/cli.py", line 339, in serve_cli
    model.serve(args)
  File "/home/curtine/git/ramalama/ramalama/model.py", line 106, in serve
    exec_cmd(exec_args)
  File "/home/curtine/git/ramalama/ramalama/common.py", line 42, in exec_cmd
    return os.execvp(args[0], args)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen os>", line 609, in execvp
  File "<frozen os>", line 651, in _execvpe
  File "<frozen os>", line 642, in _execvpe
FileNotFoundError: [Errno 2] No such file or directory
```

**Comment by ericcurtin** [comment]
Linked issue:

https://github.com/containers/ramalama/issues/167

File: ramalama/common.py [hunk 1]
```diff
@@ -38,7 +38,11 @@ def exec_cmd(args):
     if x:
         print(*args)
 
-    return os.execvp(args[0], args)
+    try:
+        return os.execvp(args[0], args)
+    except Exception:
+        print(f"os.execvp({args[0]}, {args}) failed")
```
Comment (rhatdan) [inline ramalama/common.py:44]: Print to stderror perror?

Comment (ericcurtin) [reply ramalama/common.py:44]: will change

Comment (ericcurtin) [reply ramalama/common.py]: I actually added a python perror-ish function called perror here. Like you, I tend to like these C-style functions

Comment (ericcurtin) [reply ramalama/common.py]: Wait you implemented one already, nevermind, using that

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**